wsclient/src/main/java/org/hpccsystems/ws/client/HPCCWsDFUClient.java

```diff
@@ -49,12 +51,11 @@
  */
 public class HPCCWsDFUClient extends DataSingleton
 {
-    private static URL                  originalURL;
+    private static URL originalURL;
 
     public static URL getOriginalURL() throws MalformedURLException
     {
-        if (originalURL == null)
-            originalURL = new URL(getOriginalWSDLURL());
+        if (originalURL == null) originalURL = new URL(getOriginalWSDLURL());
```

Comment:
There are a lot of format changes to legacy, in the future, let's have a separate issue for this work. Also, there seems to be a lot of combining conditionals into a single line, is that based on the eclipse format template?